plugins/fetchinvoice.c

```diff
@@ -851,6 +867,54 @@ static struct command_result *json_fetchinvoice(struct command *cmd,
 	return send_outreq(cmd->plugin, req);
 }
 
+/* FIXME: Using a hook here is not ideal: technically it doesn't mean
```

Comment:
It's a notification which doesn't block and supports multiple subscribers unlike some hooks.

Reply:
Yeah, but this gets fired before it hit the db which can cause an issue in theory.

Reply:
Ah I see the issue. But aren't notifications delivered only once we hand off to `io_loop` again? That'd mean we always commit before the notification is sent to the plugin, which is what we want.

Reply:
Ah, yes, your point about it being a notification not a hook!